code/game/machinery/cryopod.dm

```diff
@@ -455,6 +455,9 @@
 	//Eject any items that aren't meant to be in the pod.
 	var/list/items = src.contents
 	if(occupant)
+		if(usr != occupant && !occupant.client && occupant.stat<2)
```

Comment:
What is the stat greater than 2?

Reply:
Like, stat 2 is dead, do we have a special one for stuff greater than 2?

Reply:
Well, look at this one
https://github.com/discordia-space/CEV-Eris/blob/9ea20b5c4dc30d9cc0f4caaa6abbee2067b5d754/code/game/machinery/cryopod.dm#L254-L269

Reply:
Oh I'm really fucking dumb. I misread it. God I hate this stat shit sometimes.